src/cases/report_timing.cpp

```diff
@@ -64,16 +64,17 @@ int print_timing_WR(e3sm_io_config *cfg,
     MPI_Reduce(&cmeta->my_nreqs, &max_nreqs, 1, MPI_OFFSET, MPI_MAX, 0, comm);
     MPI_Reduce(&cmeta->my_nreqs, &min_nreqs, 1, MPI_OFFSET, MPI_MIN, 0, comm);
 
-    double dbl_tmp[7], max_dbl[7], min_dbl[7];
+    double dbl_tmp[8], max_dbl[8], min_dbl[8];
     dbl_tmp[0] = cmeta->pre_time;
     dbl_tmp[1] = cmeta->open_time;
     dbl_tmp[2] = cmeta->def_time;
     dbl_tmp[3] = cmeta->post_time;
     dbl_tmp[4] = cmeta->flush_time;
     dbl_tmp[5] = cmeta->close_time;
     dbl_tmp[6] = cmeta->end2end_time;
-    MPI_Reduce(dbl_tmp, max_dbl, 7, MPI_DOUBLE, MPI_MAX, 0, comm);
-    MPI_Reduce(dbl_tmp, min_dbl, 7, MPI_DOUBLE, MPI_MIN, 0, comm);
+    dbl_tmp[7] = cfg->comp_time;
```

Comment:
This additional reduction item is no longer necessary, when using sleep time directly.